src/tasks/submit/task.rs

```diff
@@ -23,6 +23,42 @@ use signet_constants::SignetSystemConstants;
 use std::{ops::Range, time::Instant};
 use tokio::{sync::mpsc, task::JoinHandle};
 
+/// Helper macro to log an event within a span that is not currently entered.
+macro_rules! span_scoped   {
```

Comment:
we have a bunch of utility macros across all of our repos and i wonder at which point we might wanna pull the most useful ones into somewhere like bin-base